assets/monitoring-plugin/config-map.yaml

```
@@ -8,8 +8,7 @@ data:
       default_type       application/octet-stream;
       keepalive_timeout  65;
       server {
-        listen              9443 ssl;
-        listen              [::]:9443 ssl;
+        listen              LISTEN_ADDR_PORT ssl;
```

just to make it clear that it's not the final value:
```suggestion
        listen              LISTEN_ADDRESS_PORT_REPLACED_AT_TIME ssl;
```

---

Yeah, it can get confusing, I'll change this

---

(I think he meant `..._AT_RUNTIME`)

---

I can change it, it wouldn't hurt and may be more clear, thanks!